### PR TITLE
refactor: tests/integration/210.command.deploy.test.cjs

### PR DESCRIPTION
**Pull request by hereje**
🎉 Thanks for submitting a pull request! 🎉

#### Summary

- refactor: move and rename file
- refactor: convert file to esm
- refactor: replace ava with vitest
- refactor: replace got with node-fetch
- style: apply lint rules

Related to: #5698 

---

For us to review and ship your PR efficiently, please perform the following steps:

- [ ] Open a [bug/issue](https://github.com/netlify/cli/issues/new/choose) before writing your code 🧑‍💻. This ensures we can discuss the changes and get feedback from everyone that should be involved. If you\`re fixing a typo or something that\`s on fire 🔥 (e.g. incident related), you can skip this step.
- [x] Read the [contribution guidelines](../CONTRIBUTING.md) 📖. This ensures your code follows our style guide and
      passes our tests.
- [x] Update or add tests (if any source code was changed or added) 🧪
- [ ] Update or add documentation (if features were changed or added) 📝
- [] Make sure the status checks below are successful ✅

**A picture of a cute animal (not mandatory, but encouraged)**

### CODE REVIEW

File: tests/integration/commands/deploy/deploy.test.mjs [hunk 1]
```diff
@@ -712,85 +736,78 @@ if (process.env.NETLIFY_TEST_DISABLE_LIVE !== 'true') {
...
-    console.log(`deleting test site "${SITE_NAME}". ${siteId}`)
-    await callCli(['sites:delete', siteId, '--force'])
+      const response = await fetch(`${deployUrl}/.netlify/functions/bundled-function-1`).then((res) => res.text())
+      t.expect(response).toEqual('Bundled at deployment')
+    })
   })
-}
+})
 
 test('always pass, used for forked PRs since ava fails when no tests are present', (t) => {
```
Comment (lukasholzer) [inline tests/integration/commands/deploy/deploy.test.mjs:811]: do we still need that test?

Comment (hereje) [reply tests/integration/commands/deploy/deploy.test.mjs:811]: Not sure. 
Should I remove it from the file?

**Review by lukasholzer — DISMISSED** [review]

**Comment by sarahetter** [comment]
Lint error in test

**Comment by sarahetter** [comment]
Another weird diff

**Review by sarahetter — APPROVED** [review]

**Pull request merged**